vrpn_Laputa.C

```diff
@@ -25,7 +25,8 @@ static const vrpn_int32 NUM_CHANNELS = 11;
 vrpn_Laputa::vrpn_Laputa(const char *name, vrpn_Connection *c)
     : vrpn_Analog(name, c)
     , vrpn_HidInterface(
-          m_filter = new vrpn_HidProductAcceptor(LAPUTA_VENDOR, LAPUTA_PRODUCT))
+          m_filter = new vrpn_HidProductAcceptor(LAPUTA_VENDOR, LAPUTA_PRODUCT),
```

Comment:
Took me a while to find a HID device that wasn't part of a family of related devices all subclassing a base - turns out the HDK is a decent example here :D  So, what you want to do is just pass the newly-created acceptor here, and don't have an `m_filter` member yourself - in your destructor you delete `m_acceptor` (which is in `vrpn_HidInterface` base class) instead where you used to delete `m_filter`.